src/Core/src/Platform/iOS/MauiImageView.cs

```diff
@@ -22,12 +23,21 @@ public MauiImageView(CGRect frame)
 		{
 		}
 
+		[UnconditionalSuppressMessage(IUIViewLifeCycleEvents.UnconditionalSuppressMessage, "MA0002")]
+		EventHandler? _movedToWindow;
```

Comment:
Can we add a explanation why supressing?

Reply:
Do we need an additional one in addition to the const? 

https://github.com/dotnet/maui/pull/16868/files#diff-f5450e00e8422d29dd1d2671fc035691ad5bdd53d086bd9b08227f708d8fa3cdR7-R8

Reply:
I thinks we are using justification as category , maybe we want something like : https://github.com/dotnet/maui/blob/01fc5eb0df4a1e5acbb947de313966e533f9ce09/src/Core/src/Platform/iOS/MauiPicker.cs#L22
```
		[UnconditionalSuppressMessage("Memory", "MA0002", Justification = IUIViewLifeCycleEvents.UnconditionalSuppressMessage)]

```

Reply:
Fixed